index.js

```diff
@@ -89,11 +89,11 @@ function transformSource (runner, config, source, map, callback) {
     ? setTimeout(function () { child.kill() }, config.timeoutMs)
     : -1
 
-  var dataBuffers = []
+  var dataBuffers = Buffer.alloc(0)
   child.stdout.on('data', function (data) {
     // Output is delimited to filter out unwanted warnings or other output
```

Comment:
Can you move comment down to the line where `new RegExp` is called pls?

Reply:
```suggestion
    // Output is delimited to filter out unwanted warnings or other output
```
This one? Sure.